config/owasp/suppressions.xml

```diff
@@ -81,4 +81,7 @@
   <suppress until="2022-08-01">
     <cve>CVE-2022-31569</cve>
   </suppress>
+  <suppress until="2022-10-01">
+    <cve>CVE-2022-38752</cve>
```

Comment:
Please remove this suppression - a fix has been deployed to master

Reply:
added it temporarily to get pipelines moving . .have removed it thanks